src/Pororoca.Desktop/Styles/Accents/Light2.xaml

```diff
@@ -0,0 +1,56 @@
+      <Color x:Key="SystemAccentColorDark2">#759009</Color>
+      <Color x:Key="SystemAccentColorDark1">#9cc00c</Color>
+      <Color x:Key="SystemAccentColor">#c4f00f</Color>
+      <Color x:Key="SystemAccentColorLight1">#cff33f</Color>
+      <Color x:Key="SystemAccentColorLight2">#dbf66f</Color>
+      <Color x:Key="SystemAccentColorLight3">#e7f99f</Color>
+      <Color x:Key="SystemErrorTextColor">Red</Color>
+      <!-- Custom colours below -->
+      <Color x:Key="InputBackgroundValidationProblemColor">#99fecada</Color>
+      <Color x:Key="SystemInvertedAccentColor">#a90a4c</Color>
+      <Color x:Key="SystemInvertedAccentColorLight1">#b50a51</Color>
+      <Color x:Key="SystemInvertedAccentColorLight2">#c10b57</Color>
+      <Color x:Key="MainWindowMenuColor">#fcfcfc</Color>
+      <Color x:Key="MainWindowBackgroundColor">#fafafa</Color>
+      <Color x:Key="MainWindowDarkerBackgroundColor">#eaeaeb</Color>
+      <Color x:Key="MainWindowSidebarColor">#eaeaeb</Color>
+      <Color x:Key="MainWindowSeparatorsColor">#eaeaeb</Color>
+      <Color x:Key="CurrentEnvironmentLabelColor">#a90a4c</Color>
+      <Color x:Key="ListBoxBackgroundColor">#eaeaeb</Color>
+      <Color x:Key="ListBoxItemSelectedBackgroundColor">#fcfcfc</Color>
+      <Color x:Key="DragAndDropItemDraggingColor">Black</Color>
+
+      <Thickness x:Key="ButtonSecondaryBorderThickness">0</Thickness>
+      <Color x:Key="ButtonSecondaryBorderBrush">Transparent</Color>
+      <Color x:Key="ButtonSecondaryHighlightBorderBrush">Transparent</Color>
+      <Color x:Key="ButtonSecondaryBackgroundColor">#eaeaeb</Color>
+      <Color x:Key="ButtonSecondaryHighlightBackgroundColor">#cff33f</Color>
+
+      <Color x:Key="WebSocketConnectedColor">#a90a4c</Color>
+      <Color x:Key="WebSocketMessageFromClientPrimaryColor">#780736</Color>
+      <Color x:Key="WebSocketMessageFromServerPrimaryColor">Green</Color>
+      <Color x:Key="DefaultIconsColor">Black</Color>
+      <Color x:Key="DefaultTextForegroundColor">#777777</Color>
+      <Color x:Key="TextEditorBackgroundColor">White</Color>
+      <Color x:Key="TextEditorForegroundColor">Black</Color>
+      <Color x:Key="ResponseCapturedValueTextColor">#a90a4c</Color>
+      <SolidColorBrush x:Key="TextControlBorderBrushFocused" Color="Black"/>
+      <SolidColorBrush x:Key="CheckBoxBorderBrushCheckedPointerOver" Color="Black"/>
+      <SolidColorBrush x:Key="CheckBoxCheckBackgroundStrokeCheckedPointerOver" Color="Black"/>
+      <SolidColorBrush x:Key="SystemControlHighlightAccentBrush" Color="#a90a4c"/>
+      <SolidColorBrush x:Key="ComboBoxItemBackgroundSelected" Color="#dbf66f"/>
+      <SolidColorBrush x:Key="ComboBoxItemBackgroundSelectedPointerOver" Color="#cff33f"/>
+      <SolidColorBrush x:Key="ComboBoxItemBackgroundSelectedPressed" Color="#c4f00f"/>
+      <SolidColorBrush x:Key="TextControlSelectionHighlightColor" Color="#dbf66f"/>
+      <!--Color x:Key="TreeViewItemTargetMovementHighlightColor">#22000000</Color-->
```

Comment:
The text controls contrasted a bit on the white background. To lower this contrast, we can use a darker tone of the sidebar color for them:

```xaml
...
<SolidColorBrush x:Key="ComboBoxBorderBrush" Color="#bebec1"/>
<SolidColorBrush x:Key="ComboBoxBorderBrushPointerOver" Color="#a4a4a8"/>
<SolidColorBrush x:Key="TextControlBorderBrush" Color="#bebec1"/>
<SolidColorBrush x:Key="TextControlBorderBrushPointerOver" Color="#a4a4a8"/>
<SolidColorBrush x:Key="TextControlBorderBrushFocused" Color="#a4a4a8"/>
<SolidColorBrush x:Key="TextControlForeground" Color="#595959"/>
<SolidColorBrush x:Key="TextControlForegroundPointerOver" Color="#4d4d4d"/>
<SolidColorBrush x:Key="TextControlForegroundFocused" Color="#4d4d4d"/>
<SolidColorBrush x:Key="ComboBoxForeground" Color="#595959"/>
<SolidColorBrush x:Key="ComboBoxForegroundFocused" Color="#4d4d4d"/>
...
```

Before:

![Captura de tela 2024-08-26 115746](https://github.com/user-attachments/assets/933a4f6d-643e-4cc0-8c10-255c1f9c15dd)

After:

![Captura de tela 2024-08-26 115703](https://github.com/user-attachments/assets/5385c6d5-1482-452c-b0ee-198fa3ec2224)

Reply:
I do agree that the contrast is too high now, but I think the colours in "After:" are having too little contrast too. So with the help of a contrast checking tool (https://www.siegemedia.com/contrast-ratio) I tried to move the lightness values around to make them more visible. Hopefully they hit a pretty appropriate spot now. 

![image](https://github.com/user-attachments/assets/4b236884-b429-4a00-8392-63e50b535179)

These are the colours I tried using, 

```Xml
<SolidColorBrush x:Key="TextControlBorderBrush" Color="#939397"/>
<SolidColorBrush x:Key="TextControlBorderBrushPointerOver" Color="#7c7c83"/>
<SolidColorBrush x:Key="TextControlBorderBrushFocused" Color="#7c7c83"/>
<SolidColorBrush x:Key="TextControlForeground" Color="#595959"/>
<SolidColorBrush x:Key="TextControlForegroundPointerOver" Color="#4d4d4d"/>
<SolidColorBrush x:Key="TextControlForegroundFocused" Color="#4d4d4d"/>
<SolidColorBrush x:Key="ComboBoxBorderBrush" Color="#939397"/>
<SolidColorBrush x:Key="ComboBoxBorderBrushPointerOver" Color="#7c7c83"/>
<SolidColorBrush x:Key="ComboBoxForeground" Color="#595959"/>
<SolidColorBrush x:Key="ComboBoxForegroundFocused" Color="#4d4d4d"/>
```

By the way, is there an easy way to theme the pointer-over colours for the sidebar elements? I feel that it would be nice if we can let it use a variant of light green, so that it can have a similar experience with the buttons on the rest of the UI.

Reply:
The contrast looks good that way.

Avalonia defines most of the brushes [here](https://github.com/AvaloniaUI/Avalonia/blob/06ece8da8813119b0feb0d35de397dd031157095/src/Avalonia.Themes.Fluent/Accents/FluentControlResources.xaml#L694)

The colors for TreeViewItem should be `TreeViewItemBackground` and `TreeViewItemBackgroundPointerOver`, they can be overridden.

Reply:
Hmm, a bit of experimenting... how does this look? 

Hover: 
![image](https://github.com/user-attachments/assets/ee89c350-40e5-49a6-87c1-a31d48063059)

Pressed: 
![image](https://github.com/user-attachments/assets/4fee451f-8f6a-49e7-81dc-0751739a5668)

Here are the colours,
```Xml
<SolidColorBrush x:Key="TreeViewItemBackgroundPointerOver" Color="#e3f989"/>
<SolidColorBrush x:Key="TreeViewItemBackgroundPressed" Color="#bae50d"/>
<SolidColorBrush x:Key="TreeViewItemBackgroundSelectedPressed" Color="#bae50d"/>
```

Reply:
Looks good!